apps/api/src/view-models/presentation/formatResourceCompositeIdentifier.ts

```diff
@@ -0,0 +1,3 @@
+import { ResourceCompositeIdentifier } from '../../domain/models/types/entityCompositeIdentifier';
+
+export default ({ id, type }: ResourceCompositeIdentifier): string => `${type}/${id}`;
```

Comment:
This is how Arango presents composite identifiers, but we don't necessarily have to follow suit. Just figured it was a good starting point.

Reply:
As long as we have access to type and id (arango "_key") separately elsewhere it should be fine.  Need to look up where you use this.